### ISSUE TITLE
when focused, there are some lines outside

### ISSUE THREAD

**Issue by JavanShen** [chr15m/DoodleCSS#16]
when focused, there are some lines outside, i think `outline: none` can fix this？
![image](https://user-images.githubusercontent.com/53335668/199011976-c3e9c3ea-2093-4eb5-98cd-c7fb6fac260d.png)

**Comment by chr15m** [comment]
Yes, we need to come up with a way to show focus that is consistent with the DoodleCSS style as per #9.
